src/classes/Carts/Cart.ts

```diff
@@ -126,10 +126,34 @@ export default abstract class Cart {
         return this.our[sku] !== undefined ? this.our[sku].amount : 0;
     }
 
+    getOurGenericCount(sku: string): number {
+        return this.getGenericCount(sku, s => this.getOurCount(s));
+    }
+
+    getGenericCount(sku: string, getCountFn: (sku: string) => number): number {
+        const pSku = SKU.fromString(sku);
+        if (pSku.quality === 5) {
```

Comment:
Do we not have an enum for all item qualities? If not, possible addition for the future? (Would result in us able to use something like `QUALITY.UNUSUAL`)

Reply:
we do not have an enum for all the qualities. I just use what's in the schema and figure that is more future proof as the schema gets updated.